tests/unit/DocumentFactoryTest.php

```diff
@@ -117,6 +120,18 @@ class DocumentFactoryTest extends SapphireTest
         ],
     ];
 
+    public static function setUpBeforeClass()
```

Comment:
Could you remove these hacks please and see if CircleCI is happy with it?

Reply:
Can you have a look here and help me on this CircleCI failed test?
https://circleci.com/gh/Firesphere/silverstripe-solr-search/518
This is before the hack that I made in order for the tests to run fixtures.

Reply:
I'll do that over the weekend. I'm not sure why it fails suddenly though :(

Reply:
Have removed the fixture hack since I got a passing tests result here: https://circleci.com/gh/Firesphere/silverstripe-solr-search/636
After that you will see the errors, I ran into on succeeding CircleCI test runs.